src/shared/codelens/codeLensUtils.ts

```diff
@@ -297,3 +358,14 @@ export function initializeTypescriptCodelens(context: ExtContext): void {
         })
     )
 }
+
+function getPathRelativeToWorkspaceFolderPath(childPath: string): string | undefined {
```

Comment:
there is `getNormalizedRelativePath()` but it doesn't do what we want here. 

This should probably live in `workspaceUtils.ts` so that it is more discoverable

Reply:
name suggestion: `getWorkspaceRelativePath` or `toWorkspaceRelativePath`